package.json

```diff
@@ -83,14 +83,16 @@
     "eslint": "^7.4.0",
     "faker": "^4.1.0",
     "grunt-bower-task": "^0.5.0",
-    "husky": "^4.3.0",
+    "husky": "^5.0.0-beta.0",
```

Comment:
I use husky 5 myself (as a supporter), but I don't think its really ready to use fore everyone, is there a reason you upgraded?

Reply:
```suggestion
     "husky": "^4.3.0",
```

Reply:
Oh sorry I was trying out on local machine for w/ versions of husky. sorry would get this removed asap